Documentation/index.rst

```diff
@@ -232,6 +234,22 @@ or in staging environments.
       sites:
         neossitename:
           overrideHostname: 'example.org'
+          pagePathPrefix: '/neosuri'
+
+Show only page information
+----------------------------------------
+
+To show the information for the whole site and not only the current page,
+the setting 'includePagePath' can be set. This removes the filter for the frontend uri from
+the API request.
+
+.. code-block:: yaml
+
+  Neos:
+    GoogleAnalytics:
+      sites:
+        neossitename:
```

Comment:
I would suggest change this to: `siteName`. Or `neosSiteName`

Reply:
@crydotsnake I updated the documentation to use neosSiteName. Could you have another look?